cartography/intel/aws/ec2/tgw.py

```diff
@@ -0,0 +1,214 @@
+import logging
+
+from .util import get_botocore_config
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
```

Comment:
For the `get_*` functions, can you please add the new `@aws_handle_regions` decorator? See https://github.com/lyft/cartography/blob/cd55965dbe24fcdc10d146328b6e460d197e4670/cartography/util.py#L68.

Reply:
will do